cmd/sip003/main.go

```diff
@@ -23,13 +23,15 @@ func main() {
 			RemoteAddr: arguments.RemoteAddr,
 			LocalAddr:  arguments.LocalAddr,
 			ServerName: options["sni"],
+			Cleartext: options["cleartext"] == "cleartext",
 		}.Run()
 	case "server":
 		impl.GunServiceServerImpl{
 			RemoteAddr: arguments.LocalAddr,
 			LocalAddr:  arguments.RemoteAddr,
 			CertPath:   options["cert"],
 			KeyPath:    options["key"],
+			Cleartext: options["cleartext"] == "cleartext",
```

Comment:
maybe you forgot to go fmt the project.
I thought it was <kbd>Ctrl+Alt+Shit+P</kbd> or something.